go/src/koding/klient/machine/mount/index/index_std.go

```diff
@@ -0,0 +1,14 @@
+// +build !darwin,!linux
```

Comment:
it seems this file is not needed, if fi.Sys returns nil, linux implementation will result with the same, if sometime in the future Sys is implemented any other OS, it will work without requiring any code change